Sources/tart/Fetcher.swift

```diff
@@ -0,0 +1,46 @@
+import Foundation
+import AsyncAlgorithms
+
+class Fetcher: NSObject, URLSessionTaskDelegate, URLSessionDelegate, URLSessionDataDelegate {
```

Comment:
From the name it seems it can be used for fetching multiple requests but I don't think `fetch` can handle it.

Reply:
Do you want to rename it? Or add an exception in `fetch()` if it was called twice?